libs/components/indicators/src/lib/modules/label/label.component.ts

```diff
@@ -53,4 +94,30 @@ export class SkyLabelComponent {
     this.baseIcon = indicatorIcon.modernThemeBaseIcon;
     this.topIcon = indicatorIcon.modernThemeTopIcon;
   }
+
+  private updateDescriptionComputed(): void {
+    if (this.descriptionType) {
+      switch (this.descriptionType) {
+        case 'none':
+          this.descriptionComputed = '';
+          break;
+        case 'custom':
+          this.descriptionComputed = this.customDescription;
+          break;
+        default:
+          this.resources
+            .getString(
+              'skyux_label_sr_' + this.descriptionType.replace(/-/g, '_')
+            )
+            .subscribe((value) => {
```

Comment:
This observable will be subscribed to every time `updateDescriptionComputed` is called. Is there a better way to handle this?

Reply:
not that I can think of--it will most likely only ever be called once, but on the off chance they change the description type we do need to update the resource string. 

I guess we could preemptively get _all_ of the description type options in `ngOnInit` and then just have the logic to switch them here....would that be preferable? 🤔